modules/python3/src/pybindutils.cpp

```diff
@@ -132,28 +133,44 @@ std::unique_ptr<BufferBase> createBuffer(pybind11::array& arr) {
     auto ndim = arr.ndim();
     ivwAssert(ndim == 1 || ndim == 2, "ndims must be either 1 or 2");
     auto df = pyutil::getDataFormat(ndim == 1 ? 1 : arr.shape(1), arr);
-    BufferFromArrayDispatcher dispatcher;
-    return dispatching::dispatch<std::unique_ptr<BufferBase>, dispatching::filter::All>(
-        df->getId(), dispatcher, arr);
+
+    if (pybind11::array::c_style == (arr.flags() & pybind11::array::c_style)) {
+        BufferFromArrayDispatcher dispatcher;
+        return dispatching::dispatch<std::unique_ptr<BufferBase>, dispatching::filter::All>(
+            df->getId(), dispatcher, arr);
+    } else {
+        throw Exception("Unable to create a Buffer from array", IVW_CONTEXT_CUSTOM("pybindutils"));
```

Comment:
MIght be a good idea to mention why here, and how to make a numpy array c_style..